test/e2e/helpers.go

```diff
@@ -533,3 +536,133 @@ func CollectArtifacts(ctx context.Context, kubeconfigPath, name string, args ...
 	fmt.Printf("stdout:\n%s\n", string(stdout.Bytes()))
 	return err
 }
+
+// Apply wraps `kubectl apply ...` and prints the output so we can see what gets applied to the cluster.
+func Apply(ctx context.Context, clusterProxy framework.ClusterProxy, resources []byte, args ...string) error {
```

Comment:
Out of curiosity - was the usage of `Apply` what unblocked e2e?

Reply:
Yes